.gitignore

```
@@ -0,0 +1,2 @@
+/.idea
+/example-pandapower/temp.json
```

I think you can change the contents of this file to the following:

```suggestion
temp.json
```

and move this file to `example-pandapower/.gitignore`. 

If you want, you can add a new `.gitignore` file to the root level to ignore the `.idea` folder.